mig/shared/griddaemons/login.py

```diff
@@ -682,7 +694,7 @@ def refresh_job_creds(configuration, protocol, username):
 
 
 def refresh_share_creds(configuration, protocol, username,
-                        share_modes=['read-write']):
+                        share_modes=[READ_WRITE_ACCESS]):
```

Comment:
I'm not a big fan of the mutable object share_modes=[] as a default argument. It's not an issue in this specific instance but it can cause issues later if any modification applied to share_modes inside of refresh_share_creds will persist across calls https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments